site2/website/package.json

```diff
@@ -20,5 +20,8 @@
   },
   "jest": {
     "preset": "jest-puppeteer"
+  },
+  "dependencies": {
+    "redoc": "^2.0.0-alpha.30"
```

Comment:
Can this be moved to devDependencies since the site is statically generated.

Reply:
yeah moved